### DISCUSSION TITLE
Add a new Save & Add button to record sets

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5618] [Feature Requests]
<img width="1238" alt="Screen Shot 2022-08-23 at 10 35 46 AM" src="https://user-images.githubusercontent.com/37256050/186200692-128550af-bfcb-49c9-93dd-2c96fe35619e.png">

> **Robert Cubey from CSIRO on [Specify Community Forum](https://discourse.specifysoftware.org/t/when-working-in-a-record-set-could-we-have-a-save-and-next-as-well-as-a-save/617):**
> When working in a record set could we have a save and next as well as a save?

**Comment by maxpatiiuk** [comment]
We have a "Save and Add" button that saves the current record and opens a new record.
Do they want to save the record, and go to the next existing record?

**Comment by grantfitzsimmons** [comment]
<img width="617" alt="image" src="https://user-images.githubusercontent.com/37256050/186222838-e6612900-77a7-4cbb-8d82-8c6d698a3272.png">

In a Collection Object record set, I am only seeing Clone, Add, and Save.

**Comment by maxpatiiuk** [comment]
Oh, right, I think we removed it.

Where is the first screenshot comming from then?

**Comment by grantfitzsimmons** [comment]
I made it

**Comment by maxpatiiuk** [comment]
Oh, I though my memory was failing me 😂
